supervisor/api/host.py

```diff
@@ -120,25 +120,25 @@ async def services(self, request):
     def service_start(self, request):
         """Start a service."""
         unit = request.match_info.get(SERVICE)
-        return asyncio.shield(self.sys_host.services.start(unit))
+        return [asyncio.shield(self.sys_host.services.start(unit))]
```

Comment:
Whenever possible I adjusted our dbus interfaces to return the actual output of the dbus method and only return an array if there is actually multiple outputs (or if the output itself is an array obviously). Whereas today they frequently returned an `Awaitable` which was the output wrapped in a list.

However the response of these 4 methods:
https://github.com/home-assistant/supervisor/blob/01e24a3e74c54c4b93e4c87196d1a5ebf8d0fb42/supervisor/dbus/systemd.py#L76-L106

Directly became response of the corresponding REST APIs. So I re-wrapped it in an array here for backwards compatibility.

Is that actually necessary though? I have no idea who or what uses these APIs but I can't imagine they care about the output. It's a systemd job ID that looks like this: `/org/freedesktop/systemd1/job/7623`. To my knowledge there is nothing a user can do with this response. There's no API they can pass this identifier to get more info, no place they can look up more info about it, they can't take the response from `start_service` and pass it `restart_service`, etc.

I was being careful but I feel like we can drop the array. Maybe we shouldn't return a response at all? Unless I'm missing something.